pokemongo_bot/cell_workers/update_live_stats.py

```diff
@@ -56,16 +56,9 @@ class UpdateTitleStats(BaseTask):
     - stardust_earned : The number of earned stardust since the bot started.
     - highest_cp_pokemon : The caught pokemon with the highest CP since the bot started.
     - most_perfect_pokemon : The most perfect caught pokemon since the bot started.
-
-    min_interval : The minimum interval at which the title is updated,
-                   in seconds (defaults to 10 seconds).
-                   The update interval cannot be accurate as workers run synchronously.
-    stats : An array of stats to display and their display order (implicitly),
-            see available stats above.
     """
     SUPPORTED_TASK_API_VERSION = 1
 
-
     def __init__(self, bot, config):
```

Comment:
Tasks shouldn't be implementing __init__. BaseTasks's __init__ calls `initialize` which tasks should implement to do most of this logic. You should be able to check out other tasks to see how they do it.